code/__DEFINES/dcs/signals/signals_mob/signals_mob_carbon.dm

```diff
@@ -79,3 +79,12 @@
 //from /mob/living/carbon/human/proc/check_shields(): (atom/hit_by, damage, attack_text, attack_type, armour_penetration)
 #define COMSIG_HUMAN_CHECK_SHIELDS "human_check_shields"
 	#define SHIELD_BLOCK (1<<0)
+
+// Mob transformation signals
+///Human transforming into a monkey
+#define COMSIG_HUMAN_MONKEYIZE "human_monkeyize"
+///Monkey transforming into a human
+#define COMSIG_MONKEY_HUMANIZE "monkey_humanize"
+///Changeling transforming
+#define COMSIG_CHANGELING_TRANSFORM "changeling_transform"
```

Comment:
This should be in its own file (signals_changeling or similar), and be clear on what inputs its taking and when this happens